data/card_definitions.json

```diff
@@ -1974,6 +1974,370 @@
+					]
+				}
+			]
+		}
+	},
+	{
+		"id": "chaos_repel",
+		"type": "special",
+		"display_name": "Repel",
+		"force_cost": 0,
+		"gauge_cost": 0,
+		"range_min": 0,
+		"range_max": 0,
+		"speed": 3,
+		"power": 5,
+		"armor": 0,
+		"guard": 0,
+		"effects": [
+			{
+				"timing": "during_strike",
+				"condition": "used_character_bonus",
+				"effect_type": "calculate_range_from_buddy",
+				"buddy_name": "card",
+				"character_effect": true
+			},
+			{
+				"timing": "during_strike",
+				"condition": "buddy_in_play",
+				"effect_type": "opponent_cant_move_past_buddy",
+				"buddy_name": "card"
+			},
+			{
+				"timing": "hit",
+				"effect_type": "push",
+				"amount": 2
+			},
+			{
+				"timing": "cleanup",
+				"condition": "buddy_in_play",
+				"effect_type": "remove_opponent_cant_move_past_buddy",
+				"buddy_name": "card"
+			}
+		],
+		"boost": {
+			"boost_type": "continuous",
+			"force_cost": 0,
+			"cancel_cost": -1,
+			"display_name": "Compounding Plots",
+			"effects": [
+				{
+					"timing": "during_strike",
+					"effect_type": "powerup",
+					"amount": 1
+				},
+				{
+					"timing": "now",
+					"effect_type": "bonus_action"
+				}
+			]
+		}
+	},
+	{
+		"id": "chaos_conceal",
+		"type": "special",
+		"display_name": "Conceal",
+		"force_cost": 0,
+		"gauge_cost": 0,
+		"range_min": 1,
+		"range_max": 2,
+		"speed": 2,
+		"power": 5,
+		"armor": 0,
+		"guard": 0,
+		"effects": [
+			{
+				"timing": "during_strike",
+				"condition": "used_character_bonus",
+				"effect_type": "calculate_range_from_buddy",
+				"buddy_name": "card",
+				"character_effect": true
+			},
+			{
+				"timing": "during_strike",
+				"effect_type": "higher_speed_misses",
+				"dodge_at_speed_greater_than": 5
+			},
+			{
+				"timing": "after",
+				"effect_type": "retreat",
+				"amount": 1
+			}
+		],
+		"boost": {
+			"boost_type": "continuous",
+			"force_cost": 2,
+			"cancel_cost": -1,
+			"display_name": "Checkmate",
+			"effects": [
+				{
+					"timing": "during_strike",
+					"effect_type": "powerup",
+					"amount": 4
+				},
+				{
+					"timing": "now",
+					"effect_type": "discard_to",
+					"amount": 1
+				},
+				{
+					"timing": "now",
+					"effect_type": "strike"
+				}
+			]
+		}
+	},
+	{
+		"NOTE": "CHAOS CARDS END",
+		"id": "chaos_thatsyourprey",
+		"type": "special",
+		"display_name": "That's Your Prey",
+		"force_cost": 0,
+		"gauge_cost": 0,
+		"range_min": 0,
+		"range_max": 1,
+		"speed": 1,
+		"power": 5,
+		"armor": 0,
+		"guard": 5,
+		"effects": [
+			{
+				"timing": "during_strike",
+				"condition": "used_character_bonus",
+				"effect_type": "calculate_range_from_buddy",
+				"buddy_name": "card",
+				"character_effect": true
+			},
+			{
+				"timing": "during_strike",
+				"condition": "not_buddy_in_play",
+				"effect_type": "armorup",
+				"amount": 2
+			},
+			{
+				"timing": "hit",
+				"condition": "range",
+				"condition_amount": 0,
+				"condition_from_source": true,
```

Comment:
maybe instead of this, condition buddy_in_opponent_space used for litchi?

Reply:
buddy shouldn't be in play unless they used their char ability

Reply:
will borrow the buddy_in_opponent_space condition; however, the buddy can also be in play because of the deep revenance boost

Reply:
oh weird, so you deep rev strike and if it's this it calculates range from yourself.
But it should still work in that case?

Reply:
yup, it still gets the place in space effects; not sure how useful it is but